docs/develop/java/workflows.mdx

```diff
@@ -0,0 +1,175 @@
+---
+sidebar_position: 7
+description: "Find out how you can implement workflows with the Restate SDK."
+---
+
+import Admonition from '@theme/Admonition';
+
+# Workflows
+
+The Workflow API let's you define workflows as a sequence of steps that gets executed durably.
+A workflow can be seen as a special type of Virtual Object with some special characteristics.
+
+In summary:
+
+- A `run` handler that executes exactly one time for each workflow instance (object).
+This is the handler which implements the workflow logic.
+- Optionally, other handlers that can be called multiple times, and which interact with the workflow.
+These handlers can be called up to 24 hours after the `run` handler has finished.
+- Some extra SDK functionality to signal workflows via Durable Promises.
+
+<CH.Scrollycoding>
+
+    Have a look at the code example to get a better understanding of how workflows are implemented:
+
+    <CH.Code lineNumbers={"true"}>
+        ```java SignupWorkflow.java
+        CODE_LOAD::java/src/main/java/develop/workflows/SignupWorkflow.java
+        ```
+    </CH.Code>
+
+    ---
+
+    ## Implement the `run` handler
+
+    Every workflow needs a `run` handler.
+    This handler has access to the same SDK features as Service and Virtual Object handlers.
+    For example, use `ctx.run` to log intermediate resuljava in Restate and avoid re-execution on replay.
+
+    <Admonition type="note" title={"Workflow retention time"}>
+        The retention time of a workflow execution is 24 hours.
+        After this timeout:
+        - Any K/V state is cleared
+        - The workflow's shared handlers cannot be called anymore
+        - The Durable Promises are discarded
+
+        The retention time can be configured via the [Admin API](/references/admin-api/#tag/service/operation/modify_service) per Workflow definition by setting `workflow_completion_retention`.
+    </Admonition>
```

Comment:
This reads pretty bad IMO in the side bar here. You also repeat it twice for the durable promises

How about you write this in the summary above or somewhere below after the code section?